### PR TITLE
Syntax and style corrections 

### PR DESCRIPTION
**Pull request by fermujica**
This article was displaying incorrect tagging for file names. 
Also updating terms such as "runtime" to "Mule runtime engine".

### CODE REVIEW

File: modules/ROOT/pages/update-workspace.adoc [hunk 1]
```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc:3]: HIDDEN TEXT EDIT

= Updating Workspaces
ifndef::env-site,env-github[]

File: modules/ROOT/pages/update-workspace.adoc [hunk 2]
```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
...
 endif::[]
 
-Anypoint Studio 7.2 and later simplifies the configuration of the `mule-artifact.json` file. This file is used when packaging your Mule application to identify the class loader configuration and settings of your Mule application, and the minimum required Mule runtime version to run your application. +
+Anypoint Studio 7.2 and later simplifies the configuration of the `mule-artifact.json` file. This file is used when packaging your Mule application to identify the class loader configuration and settings of your Mule application, and the minimum required Mule runtime engine version to run your application.
 
-Anypoint Studio includes a new version of the Mule Maven Plugin that can automatically infer some of this required metadata. This make it easier for you to keep track of all the declarations you need to make when exporting resources. +
+Anypoint Studio includes a new version of the Mule Maven Plugin that can automatically infer some of this required metadata. This make it easier for you to keep track of all the declarations you need to make when exporting resources.
 
 
 [%header%autowidth.spread,cols="a,a"]
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc:11]: HIDDEN TEXT, not edited

"minMuleVersion": "4.0.0",
  "requiredProduct": "MULE_EE",
  "classLoaderModelLoaderDescriptor": {
    "id": "mule",
    "attributes": {
      "exportedResources": []
    }
  },
  "bundleDescriptorLoader": {
    "id": "mule",
    "attributes": {}
  }
}
----
|
[source,json,linenums]
----
{
  "minMuleVersion": "4.0.0",
}

File: modules/ROOT/pages/update-workspace.adoc (hunk 1 above)
Comment (kbostrom) [reply modules/ROOT/pages/update-workspace.adoc:3]: I would like to try something new.
I know that our team uses imperative voice (in this case, Update) for tasks, but this is not a task.
However, the gerund case (in this case, Updating) sounds a bit like a task, too.
What would you think of "Workspace Updates:?

File: modules/ROOT/pages/update-workspace.adoc [hunk 3]
```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio 7.2 and later simplifies the configuration of the `mule-artifact.json` file. This file is used when packaging your Mule application to identify the class loader configuration and settings of your Mule application, and the minimum required Mule runtime version to run your application. +
+Anypoint Studio 7.2 and later simplifies the configuration of the `mule-artifact.json` file. This file is used when packaging your Mule application to identify the class loader configuration and settings of your Mule application, and the minimum required Mule runtime engine version to run your application.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]:
```suggestion
Anypoint Studio 7.2 and later simplifies the configuration of the `mule-artifact.json` file. This file is used when packaging your Mule application to identify its class loader configuration and settings, and the minimum required Mule runtime engine version to run your application.
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 4]
```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
...
 
-Anypoint Studio includes a new version of the Mule Maven Plugin that can automatically infer some of this required metadata. This make it easier for you to keep track of all the declarations you need to make when exporting resources. +
+Anypoint Studio includes a new version of the Mule Maven Plugin that can automatically infer some of this required metadata. This make it easier for you to keep track of all the declarations you need to make when exporting resources.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]: Is the plugin a Mule plugin or an Apache Maven plugin?
```suggestion
Anypoint Studio includes a new version of the Mule plugin for Apache Maven that can automatically infer some of this required metadata. This makes it easier for you to keep track of all the declarations you need to make when exporting resources.
```
OR
```suggestion
Anypoint Studio includes a new version of the Mule implementation of the Apache Maven plugin that can automatically infer some of this required metadata. This makes it easier for you to keep track of all the declarations you need to make when exporting resources.
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 5]
```diff
@@ -43,28 +43,33 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
 ----
 |===
 
-Note that this simplification is meant to reduce the number of declarations you need to keep track at development time using Anypoint Studio. A fully configured 'mule-artifact.json' is still required by the runtime when executing the packaged application. You can find the generated 'mule-artifact.json' file inside the `target` directory after packaging.
+Note that this simplification is meant to reduce the number of declarations you need to keep track at development time using Anypoint Studio. A fully configured `mule-artifact.json` is still required by Mule runtime engine when executing the packaged application. You can find the generated `mule-artifact.json` file inside the `target` directory after packaging.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]:
```suggestion
Note that this simplification is meant to reduce the number of declarations you need to keep track of at development using Anypoint Studio. A fully configured `mule-artifact.json` file is still required by Mule runtime engine when executing the packaged application. You can find the generated `mule-artifact.json` file inside the `target` directory after packaging.
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 6]
```diff
@@ -43,28 +43,33 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
...
 
-The Mule Maven plugin checks the metadata declared in 'mule-artifact.json'. If there is a value that is not declared, the Mule Maven Plugin infers it from the application code before packaging.
+The Mule Maven plugin inspects the metadata declared in `mule-artifact.json`. If there is a value that is not declared, the Mule Maven Plugin infers it from the application code before packaging. If there are other metadata declared, it uses that information to package the application.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]:
```suggestion
The plugin inspects the metadata declared in `mule-artifact.json`. If there is a value that is not declared, the plugin infers it from the application code before packaging. If other metadata is declared, the plugin uses that information to package the application.
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 7]
```diff
@@ -43,28 +43,33 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
...
 
-If there are other metadata declared, it uses that information to package the application. +
-If you choose to update to the latest Mule Maven plugin, and to keep your old descriptor files, you'll need to manually keep track of your descriptors metadata.
+If you choose to update to the latest Mule Maven plugin, and want to keep your old descriptor files, you'll need to manually keep track of your descriptors metadata.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]:
```suggestion
If you choose to update to the latest plugin and want to keep your old descriptor files, you'll need to manually keep track of your descriptor metadata.
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 8]
```diff
@@ -43,28 +43,33 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
...
 
 == Upgrading From 7.1.x to 7.2 and later
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]:
```suggestion
== Upgrading from Studio 7.1.x to 7.2 and later
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 9]
```diff
@@ -43,28 +43,33 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
...
 
 When you open Anypoint Studio 7.2 and use a workspace created with previous versions, Studio pops up a note:
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc:55]:
```suggestion
When you open Anypoint Studio 7.2 and use a workspace created with previous versions, Studio displays a note:
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 10]
```diff
@@ -43,28 +43,33 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
...
 
 image::this-766bb.png[align=center]
 
-Note that Studio lists all the projects whose configurations need updating. +
-Keep in mind that Anypoint Studio does not consider closed projects for this migration.
+Note that Studio lists all the projects whose configurations need updating and that it does not consider closed projects for this migration.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]:
```suggestion
Note that Studio lists only the projects whose configurations need updating; it does not list closed projects.
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 11]
```diff
@@ -43,28 +43,33 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
...
 
 * Update the Mule Maven Plugin version in the pom.xml file of your projects listed in this dialog.
-* Remove all declarations from the 'mule-artifact.json' files of the same listed projects. +
- This process overwrites your existing 'mule-artifact.json'. If you have other declarations in your 'mule-artifact.json' that you want to preserve (for example, secureProperties declarations, redeploymentEnabled, etc.), you should back up those files outside of your workspace, perform the migration, and then restore your old 'mule-artifact.json' files. +
-We strongly recommend not to preserve the following declarations: configs, name, requiredProduct, classLoaderModelLoaderDescriptor and bundleDescriptorLoader
+* Remove all declarations from the `mule-artifact.json` files of the same listed projects. +
+This process overwrites your existing `mule-artifact.json`. If you have other declarations in your `mule-artifact.json` that you want to preserve (for example, secureProperties declarations, redeploymentEnabled, etc.), you should back up those files outside of your workspace, perform the migration, and then restore your old `mule-artifact.json` files.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]:
```suggestion
This process overwrites your existing `mule-artifact.json` file. If you have other declarations in `mule-artifact.json` that you want to preserve (for example, secureProperties, redeploymentEnabled, and so on), you should back up those files outside of your workspace, perform the migration, and then restore your old `mule-artifact.json` files.
```

File: modules/ROOT/pages/update-workspace.adoc [hunk 12]
```diff
@@ -43,28 +43,33 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
...
+
+MuleSoft strongly recommends not to preserve the following declarations:
```
Comment (kbostrom) [inline modules/ROOT/pages/update-workspace.adoc]:
```suggestion
Do not to preserve the following declarations:
```

**Review by kbostrom — APPROVED** [review]
Here you go!

File: modules/ROOT/pages/update-workspace.adoc (hunk 4 above)
Comment (fermujica) [reply modules/ROOT/pages/update-workspace.adoc]: @kbostrom "Mule Maven Plugin" is the name of the plugin. Probably not an official name product, but that's how it's been known for years: https://docs.mulesoft.com/mule-runtime/4.2/mmp-concept

**Pull request merged**